great_expectations/data_context/store/validation_definition_store.py

```diff
@@ -53,14 +53,16 @@ def gx_cloud_response_json_to_object_dict(response_json: dict) -> dict:
 
     @override
     def serialize(self, value):
-        if self.cloud_mode:
-            data = value.dict()
-            data["suite"] = data["suite"].to_json_dict()
-            return data
-
         # In order to enable the custom json_encoders in ValidationDefinition, we need to set `models_as_dict` off  # noqa: E501
         # Ref: https://docs.pydantic.dev/1.10/usage/exporting_models/#serialising-self-reference-or-other-models
-        return value.json(models_as_dict=False, indent=2, sort_keys=True)
+        output = value.json(models_as_dict=False, indent=2, sort_keys=True, exclude_none=True)
```

Comment:
TODO: keep one main path for serialization, and if we need to keep the branching, go `model -> dict`, and then if needed, `dict -> json`